packages/vitest/src/node/mocker.ts

```diff
@@ -16,6 +16,8 @@ function resolveMockPath(mockPath: string, root: string, nmName: string | null)
     const mockFolder = resolve(root, '__mocks__')
     const files = readdirSync(mockFolder)
```

Comment:
shouldn't move the files below the if statament?

Reply:
Ou! My bad. I copy paste it wrongly from another branch